packages/node_modules/@ciscospark/internal-plugin-avatar/src/avatar.js

```diff
@@ -36,8 +36,24 @@ const Avatar = SparkPlugin.extend({
    */
   _fetchAvatarUrl(uuid, options) {
     return this.store.get({uuid, size: options.size})
-      .catch(() => this.batcher.request({uuid, size: options.size}))
-        .then((item) => this.store.add(defaults({cacheControl: options.cacheControl}, item)));
+      .catch(() => Promise.all([
+        // fetch all possible sizes of avatar and store in cache
+        this.batcher.request({uuid, size: 40}),
```

Comment:
looks like avatar tests already check that spark.request is only called once. i updated the mocked return values to make them work with my changes